### PR TITLE
GP-3 Define the entities

### PR DESCRIPTION
**Pull request by ahmad-abdelraheem**
In the PR, I have defined the entities and their attribute in a ```.md``` file. You can access the file in ```docs/Entities.md```.

### CODE REVIEW

File: Docs/Entities.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Entities
+
+### User
+- ID ```number```
+- First Name ```string(25)```
+- Last Name ```string(25)```
+- Email ```string(100)```
+- Password ```string(100)```
+- Country ```string```
```
Comment (besanalqadi) [inline Docs/Entities.md:9]: Why we need to know the user's country?

File: Docs/Entities.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Skill
+- ID ```number```
+- Title ```string(50)```
```
Comment (besanalqadi) [inline Docs/Entities.md:13]: What do you mean by title?

File: Docs/Entities.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Description ```string(5000)```
+- Type ```enum {Tech | non-Tech}```
+
+### Position
+- ID ```number```
```
Comment (besanalqadi) [inline Docs/Entities.md:18]: Please clarify the position, domain and roadmap. what is the difference ?

File: Docs/Entities.md (hunk 1 above)
Comment (ahmad-abdelraheem) [reply Docs/Entities.md:9]: It's for a future planning to get analysis from their region only (Jobs analysis).

File: Docs/Entities.md (hunk 2 above)
Comment (ahmad-abdelraheem) [reply Docs/Entities.md:13]: It's basically the name, such as JavaScript, Angular and so on...

File: Docs/Entities.md (hunk 3 above)
Comment (ahmad-abdelraheem) [reply Docs/Entities.md:18]: Sure,

1. Domain : it's this general domain that's involve many positions (such as Web Development).
2. Position : it's the job position that user plan to work (such as Front-End and Back-End it's all in web development domain).
3. Roadmap : it's the ordered skills that user should take to achieve a specific position. _Keep in mind that each position may have more than one roadmap, but in our project we have one if nothing changed in out plan_.

**Review by besanalqadi — APPROVED** [review]

**Pull request merged**